### DISCUSSION TITLE
[WASM Bootstrap 4.0.0-dev.174] Exception thrown during build System.IO.DirectoryNotFoundException

### DISCUSSION THREAD

**Discussion by FrozDark** [unoplatform/uno#9481] [Q&A] [answered]
### Current behavior

Throws an exception System.IO.DirectoryNotFoundException Could not find path ...\UnoAppTest.Wasm\obj\Debug\net6.0\workAot\linker-out

### Expected behavior

Success build

### Workaround

Create that folder during build as it clears out everything everytime you start build

### Works on UWP/WinUI

Yes

### NuGet package version(s)

Uno.Wasm.Bootstrap 4.0.0-dev.174

### Affected platforms

WebAssembly

### IDE

Visual Studio 2022

### IDE version

17.3.0

**Comment by jeromelaban** [comment]
Thanks for the report! Can you provide a binlog of your build: https://platform.uno/docs/articles/uno-builds-troubleshooting.html?

**Comment by FrozDark** [comment, marked as answer]
Here it is
[msbuild.zip](https://github.com/unoplatform/uno/files/9320843/msbuild.zip)

**Reply by jeromelaban** [reply]
Thanks. This is https://github.com/unoplatform/Uno.Wasm.Bootstrap/issues/542. Disabling the linker does not work when AOT, Skia or any native binary is used.

Try setting `<WasmShellILLinkerEnabled>true</WasmShellILLinkerEnabled>`.
